sqlserver/datadog_checks/sqlserver/data/conf.yaml.example

```diff
@@ -332,6 +332,14 @@ instances:
         #
         # collection_interval: 10
 
+    ## @param stored_procedure_characters_limit - integer - optional - default: 500
+    ## Limit the number of characters of the text of a stored procedure that is collected.
+    ## The characters limit is applicable to both query metrics and query activity.
+    ## Increasing this value will increase the amount of data collected from the database,
+    ## and increase the amount of data sent to Datadog.
```

Comment:
`Please be aware that increasing this value may affect performance, as more data will be gathered from the database and sent to Datadog.`
^ if the note about performance is technically accurate, then I vote we say this @lu-zhengda. thoughts?

Reply:
make sense